api/swagger.yaml

```diff
@@ -2375,6 +2912,70 @@ definitions:
     required:
       - type
 
+####### Sensors #######
+
+  sensor:
+    type: object
+    description: Sensor
+    properties:
+      id:
+        type: string
+        format: uri
+        description: |
+          Unique ID for this `Sensor`. A URI can be assigned by the client or will be generated by the server if the client doesn't provide it.
+          Once assigned, the URI can't be changed. See [Unique Identifiers](/#section/Technical-notes/Unique-identifiers) for more information.
+        example: https://api.opentripmodel.org/api/public/v4/sensors/6666f00c-1332-472c-aff9-bc11b3d53296
+      name:
+        type: string
+        description: |
+          Name of the `Sensor`, mainly used for display and search purposes.
+        example:
+          "Temperature sensor in trailer x"
+      placement:
+        type: string
+        description: |
+          Sometimes more than one sensor can be associated with a single entity. This is the case e.g. in cooled trailers that are devided into
+          compartments with different temperatures, where each compartment has its own sensor. The `placement` member can be used to identify where a
+          sensor is placed. Parties using OpenTripModel to exchange sensor data may wish to agree on a standardized naming, but this is too specific
+          to describe in the standard.
+        example:
+          "Compartment 1"
+      externalIds:
+        type: array
+        items:
+          $ref: "#/definitions/idReference"
+      sensorType:
+        $ref: "#/definitions/sensorType"
+
+  sensorType:
+    type: object
+    description: The type of sensor
+    properties:
+      type:
+        type: string
+        enum:
+          - temperature
+          - pressure
+          - speed
+          - humidity
+          - accelerometer
+          - current
+          - voltage
+          - weight
+          - unknown
+          - other
+      other:
+        type: string
+        description: |
+          Name of the sensor type if `other` is chosen in the `type` field.
+        example: smurf sensor
```

Comment:
I think Gargamel will start using OTM too if he sees this :stuck_out_tongue:

Reply:
:')